### DISCUSSION TITLE
Love Bar / Love'O'Meter

### DISCUSSION THREAD

**Discussion by Rina14** [JirkaDellOro/FUDGE_Story#8] [Ideas]
1. Skala, die, bei den vom Autor vorbestimmten Antworten, getriggert wird.

2. Sie kann sich aufgrund der gewählten Antwort des Spielers aus den Antwortmöglichkeiten (Dialogelement), füllen oder leeren, wodurch sich die Beziehung und das Verhältnis zu einer Spielfigur verändern.

3. Das heißt, es soll durch einen zusätzlichen Faktor/Parameter, der den Antwortmöglichkeiten hinzugefügt werden kann, ermöglicht werden, dass dieser einen direkten Einfluss auf die Skala (Love Bar) hat.

4. Die Skala ist zu Beginn bis zu einem bestimmten Wert gefüllt (so muss der Spieler versuchen passende Antworten auszuwählen, um das Volumen der Love Bar zu erhöhen).

5. Ist das Skala-Volumen unter/über der Hälfte, verändert sich der Dialog mit der Figur dementsprechend.

6. Der Wert muss also gespeichert, berechnet und darauf zugegriffen werden können, um zu bestimmen, in welche Richtung sich der Dialog zu verändern hat (vielleicht durch Alternativantworten/-reaktionen der Spielfiguren, die bei den Dialogen definiert und durch Konditionen zum Wert abgefragt und anschließend angezeigt werden?). Die jeweiligen Wunsch-Werte sollten individuell vom Autor festgelegt werden können, sodass dieser freie Kontrolle über die Skala hat. 

7. Negativwerte sind i.d.R. nicht möglich, d.h. 0 ist der kleinste Wert. Es gibt zudem auch einen maximalen Wert, der nicht überschritten werden kann.

**Comment by JirkaDellOro** [comment]
Ich habe jetzt ein allgemeines UserInterface-System eingebaut, es ist bei mir im Test zu finden und funktioniert so weit. Wenn Du kannst erkläre ich es dir, aber es ist extrem simpel zu nutzen. Sämtliche Love'o'Meter-Funktionen sind damit abbildbar, denke ich, sollten aber im Detail dann von den Autoren implementiert werden.

**Reply by JirkaDellOro** [reply]
Es muss getestet werden, ob Save/Load noch funktioniert
